Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -128,6 +128,30 @@ public extension Node where Context: HTMLLinkableContext {
     }
 }
 
+public extension Attribute where Context: HTMLLinkableContext {
+    /// Assign a hreflang string to this element.
+    /// - parameter hreflang: The hreflang of the link to assign.
+    static func hreflang(_ hreflang: String) -> Attribute {
+        Attribute(name: "hreflang", value: hreflang)
+    }
+}
+
+public extension Attribute where Context: HTMLLinkableContext {
+    /// Assign an icon sizes string to this element.
+    /// - parameter sizes: The icon sizes of the link to assign.
+    static func sizes(_ sizes: String) -> Attribute {
+        Attribute(name: "sizes", value: sizes)
+    }
+}
+
+public extension Attribute where Context: HTMLLinkableContext {
+    /// Assign an icon color string to this element.
+    /// - parameter color: The icon color of the link to assign.
+    static func color(_ color: String) -> Attribute {
```

Comment:
Same comment here as above, and these two extensions can be merged into one.

Reply:
Updated.